lib/xandra/connection.ex

```diff
@@ -166,43 +175,41 @@ defmodule Xandra.Connection do
     }
 
     force? = Keyword.fetch!(options, :force)
-    transport = state.transport
 
     case prepared_cache_lookup(state, prepared, force?) do
       {:ok, prepared} ->
+        :telemetry.execute([:xandra, :prepared_cache, :hit], %{query: prepared})
```

Comment:
**Issue**: I think we don't want any measurements here, and we rather want the query to me *metadata*, no?

We should also add the connection name, host, and port here too, I think.

Reply:
Again, was described so in https://github.com/lexhide/xandra/issues/283. 

As to the connection details in cache, I'm not that sure about notifying that as well as this is only about the cache and not much to do with the connection. The `handle_prepare` function gets even called from the caller process of `Xandra.prepare`. So I'm not sure how useful that would be.

Reply:
Alright, I handled the `metadata` identical to the other events, I guess the user can discard whatever they don't think is not neccessary.

Reply:
Exactly, I think having the same metadata in most places is handy, even if it's going to be discarded.